### ISSUE TITLE
[Units] Add a Desert Lion/Wild Cat Unit (Please?)

### ISSUE THREAD

**Issue by knyghtmare** [wesnoth/wesnoth#7186]
## Context

The current selection of monster fauna caters to a wide variety of environments/biomes but somewhat lacks for desert/arid places. Would be nice if a wild-cat/desert-lion can be added to the core selection of monster units.

## Suggested Stats
HP: 52
Level 2
XP 100
(no advancements)
MP 6

### Attacks
claws 8 - 4 melee blade (marksman)
fangs 12 - 2 melee pierce

### Movement
Movement type should be adapted to desert movement and possibly either forest or hills movement

### Appearance
Maybe like a big tiger? Or something like a sabretooth?

**Comment by knyghtmare** [comment]
@Jonathan-Kelly 
> Any forest-dwelling ambush-predator big cat would be a welcome replacement for this aging beast. A tiger, lion, jaguar, leopard, panther, etc... anything level 2 or 3 with the ambush ability would be great.

> The project and UMC community might benefit most from having a tiger or lion in core, since they can be found in a wide range of climates from very cold to very hot.

I saw you discussing about wild cat units. I wanted to bring to your attention this issue I opened almost 1.5 years ago.

**Comment by CelticMinstrel** [comment]
I'll repeat that "panther" would also be a reasonable name for such a unit, as it's a generic word which can describe almost any large cat.

I don't think a _large_ cat / panther makes a whole lot of sense for a desert unit, mind you. A _smaller_ cat might fit in though.

**Comment by knyghtmare** [comment]
> I don't think a large cat / panther makes a whole lot of sense for a desert unit, mind you. A smaller cat might fit in though.

Yes, a smaller level 1 cat advancing to a level 2 wild cat/panther. Would not mind another one.

**Comment by CelticMinstrel** [comment]
Advancements wouldn't really make sense in this case, though optional ones like the wolves have would be fine I guess.

**Comment by Jonathan-Kelly** [comment]
> I wanted to bring to your attention this issue I opened almost 1.5 years ago.

I remember; gave it a "thumbs up" emoticon back then. The `marksman` special on the claws seems like a nice idea. Definitely would like to see the `ambush` ability.

**Comment by CelticMinstrel** [comment]
> The `marksman` special on the claws seems like a nice idea.

This'll just end up recanting the dunefolk argument.

> Definitely would like to see the `ambush` ability.

What exactly does this mean?

**Comment by Jonathan-Kelly** [comment]
> This'll just end up recanting the dunefolk argument.

Though the dragonfly and its advancement already have marksman special on a melee attack, if you are referring to "melee marksman" sounding odd to some.

> What exactly does this mean?

Whichever species of Panthera a new sprite ends up resembling, it needs to have the `ambush` ability (invisible in forest) if it is meant to replace the old looking leopard sprites in WoF. Ambush may or may not be an odd ability for a lion in particular, though once there is one animated panther sprite it might be trivial to create variants which resemble others in the genus (and have different abilities).

**Comment by knyghtmare** [comment]
> Whichever species of Panthera a new sprite ends up resembling, it needs to have the ambush ability (invisible in forest) if it is meant to replace the old looking leopard sprites in WoF. Ambush may or may not be an odd ability for a lion in particular, though once there is one animated panther sprite it might be trivial to create variants which resemble others in the genus (and have different abilities).

Okay, we can add ambush to it..and maybe it more green-looking. Like some vine panther..."Vine Panther"! That's the name of this unit!

Also, would any of you take this idea to doofus-01 to convert the ideation to a real unit? I think he will check github/forums this saturday/sunday

**Comment by CelticMinstrel** [comment]
> if you are referring to "melee marksman" sounding odd to some.

It's not just sounding wrong, it's simply incorrect, and translations exacerbate this.

Clearly we _do_ need to reopen the discussion if the dragonfly already added i.

> Ambush may or may not be an odd ability for a lion in particular

Ambush in a general sense probably isn't an odd ability for a lion, but I suppose I'd expect them to be ambushing from long grass (which doesn't exist in Wesnoth!) rather than forest.

> maybe it more green or grey looking.

…why green?

> Whichever species of Panthera

Doesn't need to be Panthera. The term "panther" can also refer to a puma/cougar, which is not in Panthera.
